src/main/resources/templates/test.ftl

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <#include "./partials/head.ftl" />
+    <title>Database Test</title>
+</head>
+<body>
+    <#include "./partials/navbar.ftl" />
+    <h1>Database Test</h1>
+
+    <!-- DEBUG-->
+    <!-- <p>This example demonstrates how to assign an "onclick" event to a p element.</p> -->
+    <p></p>
+    <p id="demo" onclick="myFunction()">Click me.</p>
```

Comment:
This random button confused me at first because it didn't seem to have an associated action when I tried using it. I'd suggest removing it since it provides no additional functionality to this Database Test.